src/app/core/services/registration-form-configuration/registration-form-configuration.service.ts

```diff
@@ -167,7 +169,7 @@ export class RegistrationFormConfigurationService {
   }
 
   cancelRegistrationForm(config: RegistrationConfigType): void {
-    config.sso ? this.accountFacade.cancelRegistration() : this.router.navigate(['/home']);
+    config.sso ? this.accountFacade.cancelRegistration() : this.router.navigate([config.cancelUrl || '/home']);
```

Comment:
I'd rather go for the [Nullish coalescing operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator)